dropwizard-zipkin/src/main/java/smartthings/dw/zipkin/ZipkinModule.java

```diff
@@ -33,43 +33,55 @@ public ZipkinModule(ZipkinConfiguration config) {
     protected void configure() {
         Endpoint endpoint = getHostEndpoint();
         Reporter<Span> reporter = config.getReporter().build();
-        Brave brave = new Brave.Builder(
-            endpoint.ipv4,
-            config.getServicePort(),
-            config.getServiceName()
-        )
-            .reporter(reporter)
-            .traceSampler(BoundarySampler.create(config.getSampleRate()))
+
+        Tracing tracing = Tracing.newBuilder()
+            .endpoint(endpoint)
+            .localServiceName(config.getServiceName())
+            .spanReporter(reporter)
+            .sampler(BoundarySampler.create(config.getSampleRate()))
             .traceId128Bit(config.isTraceId128Bit())
             .build();
 
         bind(new TypeLiteral<Reporter<Span>>() {
-        })
-            .toInstance(reporter);
-        bind(Brave.class).toInstance(brave);
+        }).toInstance(reporter);
+
+        bind(Tracing.class).toInstance(tracing);
         bind(Endpoint.class).toInstance(endpoint);
     }
 
+    @Provides
+    @Singleton
+    HttpTracing providesHttpTracing(Tracing tracing) {
+        return HttpTracing.create(tracing);
```

Comment:
Not strictly necessary but I would highly encourage making clientSampler and serverSampler configurable.  Mostly because it allows you to remove noise or increase sampling for low traffic endpoints.

https://github.com/openzipkin/brave/tree/master/instrumentation/http#sampling-policy

I know if I was building a service I would for sure want to exclude `/health` for instance.